external-dependencies/README.rst

```diff
@@ -164,8 +112,8 @@ function is called by our main ``ThreeDiToolbox/__init__.py``:
   qgis-bundled or in the profile directory.
 
 - If something is missing, it calls python3's build-in "pip" to install it
```

Comment:
`pip` wordt toch niet meer gebruikt?

Reply:
In deze versie doen we het nog op de oude manier.